docs/kratos/self-service/flows/account-recovery-password-reset.mdx

```diff
@@ -108,58 +139,67 @@ To specify a trait of the identity to be used for recovery, use the following id
  }
 ```
 
-## Email templates
+### Email templates
+
+Ory Identities (Ory Kratos) comes with default email templates for recovery flows.
 
-Ory Identities (Ory Kratos) provides a set of default email templates for the recovery flows.
+You can quickly replace the defaults and customize the messages to match the look and feel of your solution. Read
+[this document](../../emails-sms/custom-email-templates) to learn more.
```

Comment:
IMO, "this document" is not very descriptive and can be confusing if you're just skimming the page to find the link to the "custom email templates" page, for example.

I'd prefer the actual page name somewhere in there, but that's just my two cents from the perspective of an avid documentation reader. ;)

Reply:
You really wouldn't skim the doc for this link specifically. This is just a comment that you can "learn more by reading something else". I prefer not using document titles in hyperlinks as it introduces wonkiness due to capitalization and makes the sentences unnaturally long. Also, I've been doing this across many docs that I've updated so this is also for consistency. Please keep it like this.

Reply:
Sure, this is just some general feedback, on stuff I noticed when trying to familiarize myself with Ory through the docs. We are, after all, writing this for developers and developers don't like reading a lot of text. So, the better "skimmable" a text is, the better, IMO. This is not blocking for me, resolving.